release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2584,6 +2584,10 @@ To work around this issue, before you update to {product-title} {product-version
 
 * There is a disk discovery delay when attaching storage to workloads. (link:https://issues.redhat.com/browse/OCPBUGS-11149[*OCPBUGS-11149*])
 
+* When upgrading from {product-title} 4.12 to 4.13, Mellanox NIC renames SR-IOV network node policies such as `ens7f0` to `ens7f0np0`. Renaming the network node policies causes the `SriovNetworkNodePolicy` object to not be able to locate the original interface. Consequently, virtual functions (VFs) cannot be created because the interface cannot be found.
```

Comment:
Should NIC be written out since this content is not near the networking content?
```suggestion
* When upgrading from {product-title} 4.12 to 4.13, the Mellanox NIC renames SR-IOV network node policies such as `ens7f0` to `ens7f0np0`. Renaming the network node policies prevents the `SriovNetworkNodePolicy` object from locating the original interface. Consequently, virtual functions (VFs) cannot be created because the interface cannot be found.
```